distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -794,4 +821,33 @@ protected boolean isCatalogEmpty() {
       return false;
     }
   }
+
+  /**
+   * Helper Option class to allow interpolation of {@code karaf.home} directory based on the
+   * provided {@link #UNPACK_DIRECTORY}.
+   */
+  static class HomeAwareVmOption extends VMOption {
```

Comment:
Can we leave a comment mentioning why this was made and the edge case it's addressing? I don't anyone else should really ever be using this.

Reply:
Considering it's itest code and does have a short javadoc, I don't know we need too much more. I doubt anyone would duplicate this approach.